javascript/ql/src/change-notes/2022-09-05-second-order-command-injection.md

```diff
@@ -0,0 +1,6 @@
+---
+category: newQuery
+---
+* Added a new query, `js/second-order-command-line-injection`, to detect shell
+  commands that may execute arbitrary code when the user has control over 
+  the arguments to a command-line program.
```

Comment:
I'd add something about which command names are currently recognized by the query, for example
> This currently flags up unsafe invocations of `git` and `hg`.